qhub_ops/template/{{ cookiecutter.repo_directory }}/image/Dockerfile.jupyterlab

```diff
@@ -34,5 +44,21 @@ RUN groupadd --gid ${NB_UID} ${NB_USER} && \
     --uid ${NB_UID} \
     ${NB_USER}
 
+RUN chown -R ${NB_UID}:${NB_UID} /opt/conda/
+
+# ========== jupyterlab install ============
+COPY jupyterlab /opt/jupyterlab
+
+RUN /opt/scripts/install-apt.sh                  /opt/jupyterlab/apt.txt \
```

Comment:
Im confused by the need to reorder the install steps. Is there a reason for this?

Reply:
Yeah , if I install jupyter lab before creating  home for user, conda is not initialized. I tried not to reorder this step but it was messing up with initializing conda.